src/data/data.js

```diff
@@ -1,10 +1,10 @@
-import feed from './feed/feed';
+import { coinbase, quandl } from 'd3fc-financial-feed';
```

Comment:
Not a big deal but I think you can `import * as feed from 'd3fc-financial-feed'`